### DISCUSSION TITLE
Running Room Assistant on an old phone?

### DISCUSSION THREAD

**Discussion by WesleyDMartin** [mKeRix/room-assistant#1209] [General]
Hello! Just wondering if anyone has tried running room assistant on an old phone?
If I can jailbreak it and run straight linux, I could at least get the service running. From there I should be able to access the BT and even potentially motion and sound sensors?

It seems like it has potential, but to be honest I have not been able to install RA on anything yet, as I haven't been able to track down hardware, so I am totally net new. 

If anyone has tried it, or can think of a good reason that it wouldn't work, please let me know! If I carry on down this road, Ill be sure to come back and update what I find.

**Comment by WesleyDMartin** [comment]
Well so far I was able to get it up and running, but I ran into an issue when adding bluetooth. Looks like it depends on an @mkerix/noble which doesn't seem to work on an ARM architecture. Not sure if anyone knows how to compile something specifically for ARM, this is new territory for me.

**Reply by sudoxreboot** [reply]
This is what I came here to do lol. Following.
